students/mariusz_michalczyk/lesson_01_basics/Sum_of_digits.py

```diff
@@ -0,0 +1,8 @@
+x = int(input("Enter integer: "))
+
+suma = 0
+for i in str(x):
+    suma += int(i)
```

Comment:
That should work without string conversion. Also there was an assumption about three digits, so you could unfold that loop:
```
while x:
  suma += x % 10
  x //= 10
```